server/moodle/mod/livequiz/classes/models/livequiz.php

```diff
@@ -81,7 +81,7 @@ class livequiz {
      * @param int $timecreated
      * @param int $timemodified
      */
-    private function __construct(
+    public function __construct(
```

Comment:
Hvorfor er denne lavet public i stedet for private? Vi skal vel aldrig selv oprette et livequiz instance?

Reply:
i just tried running it, and it works when you change it to private. So there probably is a good reason @ChristofferGamel changed everything to public. (yes i used git blame 😎 )

Reply:
This wasnt the intention. Good observation 👍